Nightmare/Locales/ptBR.lua

```diff
@@ -1,42 +1,42 @@
 local L = BigWigs:NewBossLocale("Cenarius", "ptBR")
 if not L then return end
 if L then
-	--L.forces = "Forces"
-	--L.bramblesSay = "Brambles near %s"
+	L.forces = "Forças do pesadelo"
+	L.bramblesSay = "Espinheiras perto de %s"
 end
 
 L = BigWigs:NewBossLocale("Elerethe Renferal", "ptBR")
 if L then
-	--L.isLinkedWith = "%s is linked with %s"
-	--L.yourLink = "You are linked with %s"
-	--L.yourLinkShort = "Linked with %s"
+	L.isLinkedWith = "%s esta linkado com %s"
+	L.yourLink = "Você esta linkado com %s"
+	L.yourLinkShort = "Linkado com %s"
 end
 
 L = BigWigs:NewBossLocale("Il'gynoth", "ptBR")
 if L then
-	--L.custom_off_deathglare_marker = "Deathglare Tentacle marker"
-	--L.custom_off_deathglare_marker_desc = "Mark Deathglare Tentacles with {rt6}{rt5}{rt4}{rt3}, requires promoted or leader.\n|cFFFF0000Only 1 person in the raid should have this enabled to prevent marking conflicts.|r\n|cFFADFF2FTIP: If the raid has chosen you to turn this on, having nameplates enabled or quickly mousing over the spears is the fastest way to mark them.|r"
+	L.custom_off_deathglare_marker = "Marcador de Tentáculo Fulgor da Morte"
+	L.custom_off_deathglare_marker_desc = "Marca Tentáculo Fulgor da Morte com {rt6}{rt5}{rt4}{rt3}, requer assistente ou líder.\n|cFFFF0000Apenas 1 pessoa no raide deve ter isto ativado para prevenir conflitos na marcação.|r\n|cFFADFF2FDICA: Se o raide escolheu você para isso, ter placas de identificação ativadas ou passar rapidamente o mouse sobre os tentáculos é a maneira mais rápida de marcá-los.|r"
 
-	--L.bloods_remaining = "%d |4Blood:Bloods; remaining"
+ 	L.bloods_remaining = "%d |4Sangue:Sangues; faltando"
 end
 
 L = BigWigs:NewBossLocale("Emerald Nightmare Trash", "ptBR")
 if L then
-	--L.gelatinizedDecay = "Gelatinized Decay"
-	--L.befouler = "Taintheart Befouler"
-	--L.shaman = "Dire Shaman"
+	L.gelatinizedDecay = "Decomposição Gelatinizada"
+	L.befouler = "Conspurcador Cordismáculo"
+	L.shaman = "Xamã Atroz"
 end
 
 L = BigWigs:NewBossLocale("Ursoc", "ptBR")
 if L then
-	--L.custom_off_gaze_assist = "Focused Gaze Assist"
-	--L.custom_off_gaze_assist_desc = "Show raid icons in bars and messages for Focused Gaze. Using {rt4} for odd, {rt6} for even soaks. Requires promoted or leader."
+	L.custom_off_gaze_assist = "Assistente de Olhar Concentrado"
+	L.custom_off_gaze_assist_desc = "Mostra marcadores de alvo nas barras e mensagens para Olhar Concentrado. Marca o alvo com {rt4} em casts ímpares e {rt6} em casts pares. Requer assistente ou líder."

```

Comment:
Who was marking them with exactly rt4/6 then? I even saw this conflict with DBM, sometimes the skull icon remained on target but sometimes the rt4 or 6 remained, i thought it was BW